app/javascript/components/Navbar/index.js

```diff
@@ -7,71 +7,59 @@ import UnloopLogo from 'images/unloop_logo.png';
 import styles from './styles';
 import { apiGet } from '../../utils/axios';
 
-class Navbar extends React.Component {
-  constructor() {
-    super();
-    this.logout = this.logout.bind(this);
-    this.navigateToHomepage = this.navigateToHomepage.bind(this);
-  }
+function Navbar({ classes }) {
+  const navigateToHomepage = () => {
+    const homepagePath = '/';
+    window.location.href = homepagePath;
+  };
 
-  logout() {
+  const logout = () => {
```

Comment:
You don't necessarily have to set this as a const value because it's a function.